controllers/secretshare_controller.go

```diff
@@ -61,8 +61,8 @@ type SecretShareReconciler struct {
 // +kubebuilder:rbac:groups=,resources=secret;configmap,verbs=get;update;patch;create;delete;list;watch
 
 // Reconcile reads that state of the cluster for a SecretShare object and makes changes based on the state read
-func (r *SecretShareReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-	ctx := context.Background()
+func (r *SecretShareReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	ctx = context.Background()
```

Comment:
I get a warning for overwriting the variable at the beginning here. Should I remove this ctx = context.Background() line? I'm not completely sure how this works so don't want to introduce a regression

Reply:
never mind, this break the linter so will remove